python/sandbox/simple_import/example_04_numpy_simple_import.metta

```diff
@@ -0,0 +1,11 @@
+!(import! &self simple_import)
+
+!(import numpy)
+!(import numpy)
```

Comment:
just wonder why are there two `!(import numpy)`?

Reply:
Only to demonstrate that it does not cause any problem.

Reply:
Everything, including numpy.linalg.inv works after import numpy. My point was that with simple import it is more common to import something twice, because of submodules in python. So I wanted to make sure it would not cause problems in this implementation. It doesn't.